apps/backend/src/datasources/postgres/RedeemCodesDataSource.ts

```diff
@@ -38,14 +38,16 @@ export default class PostgresRedeemCodesDataSource
   async updateRedeemCode(
     redeemCode: Partial<RedeemCode> & Pick<RedeemCode, 'code'>
   ): Promise<RedeemCode> {
-    await database('redeem_codes').update({
-      code: redeemCode.code,
-      placeholderUserId: redeemCode.placeholderUserId,
-      createdBy: redeemCode.createdBy,
-      createdAt: redeemCode.createdAt,
-      claimed_at: redeemCode.claimedAt,
-      claimed_by: redeemCode.claimedBy,
-    });
+    await database('redeem_codes')
+      .update({

```

Comment:
I believe that **code, placeholderUserId, createdBy and createdAt** properties can be removed.

Reply:
removed the unnecessary attributes, thanks